### PR TITLE
➕  Add Sytellint

### PR DESCRIPTION
**Pull request by mrana17**
- `npm install --save-dev stylelint stylelint-config-standard`
- Create a `.stylelintrc.json`
- `npm install --save-dev stylelint-config-prettier`
- add to stylelintrc.json `"stylelint-config-prettier"`
- add to `scripts`:
  `"stylelint-check": "stylelint-config-prettier-check"`
  `"stylelint": "stylelint \"**/*.css\""`

### CODE REVIEW

**Review by christophvogel — APPROVED** [review]
GJ 👍

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "build": "next build",
     "start": "next start",
     "prettify": "prettier --write .",
-    "eslinter": "eslint ."
+    "eslinter": "eslint .",
+    "stylelint": "stylelint \"**/*.css\"",
+    "stylelint-check": "stylelint-config-prettier-check"
```
Comment (phlgr) [inline package.json]: Not sure how useful this script is.

**Review by phlgr — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (m-wiemers) [reply package.json]: The boy needs fresh air

**Review by m-wiemers — APPROVED** [review]
gj 🚀

**Review by cboenke — CHANGES_REQUESTED** [review]
Good job but maybe add user-guides in description for later:
- https://stylelint.io/user-guide/get-started
- https://github.com/prettier/stylelint-config-prettier

**Pull request merged**